examples/incremental_learning/helmet_detection/README.md

```diff
@@ -207,16 +207,19 @@ spec:
   outputDir: "/output"
 EOF
 ```
+
 1. The `Dataset` describes data with labels and `HE_SAVED_URL` indicates the address of the deploy container for uploading hard examples. Users will mark label for the hard examples in the address.
 2. Ensure that the path of outputDir in the YAML file exists on your node. This path will be directly mounted to the container.
 
-
 ### Check Incremental Learning Job
-query the service status
+query the service status  
+
 ```
 kubectl get incrementallearningjob helmet-detection-demo
 ```
-In the `IncrementalLearningJob` resource helmet-detection-demo, the following trigger is configured:
+
```

Comment:
Many blank newline added in this example README file. I guess this is for render issue.

Then you add an extra rule: Adding blank newline before/after **```** in every markdown files, 
It would cause more burden for markdown authors: easy to miss that, and how can they check that.

Some ways can be used to ease this problem:
- Fix render engine
- Modify the markdown files before submitting docs to the render engine.
- Place this rule into document

Reply:
No more suitable one has been found at this time